doc/classes/CollisionObject2D.xml

```diff
@@ -269,6 +269,9 @@
 		<member name="input_pickable" type="bool" setter="set_pickable" getter="is_pickable" default="true">
 			If [code]true[/code], this object is pickable. A pickable object can detect the mouse pointer entering/leaving, and if the mouse is inside it, report input events. Requires at least one [code]collision_layer[/code] bit to be set.
 		</member>
+		<member name="on_disable" type="int" setter="set_on_disable" getter="get_on_disable" enum="CollisionObject2D.DisableMode" default="0">
```

Comment:
Wouldn't `disable_mode` be a clearer name? `on_disable` sounds like a signal callback.

Reply:
I second disable_mode probably being best

Reply:
Yes, this name does sound better! Just pushed with this change.